src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/AsyncEventHandler_OnElement_ActionEventArgs_Lambda/TestComponent.mappings.txt

```diff
@@ -1,15 +1,25 @@
-﻿Source Location: (1:0,1 [28] x:\dir\subdir\Test\TestComponent.cshtml)
-|using System.Threading.Tasks|
-Generated Location: (318:11,0 [28] )
-|using System.Threading.Tasks|
+﻿Source Location: (1:0,1 [6] x:\dir\subdir\Test\TestComponent.cshtml)
+|using |
```

Comment:
I would expect this change to break tooling, as the usings aren't being mapped properly anymore.

Reply:
Email reply was garbage on this...

How does this break tooling? The `using` is mapped, and then the `System.Threading.Tasks` is also mapped

Reply:
Please disregard as 'user can't read'.

I didn't notice the actual namespace underneath.